src/test/java/DataAccessTest/BankingDataProcessTest.java

```diff
@@ -0,0 +1,42 @@
+package DataAccessTest;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import com.aviva.DataAccess.BankingDataProcess;
+import com.aviva.Entities.AccountHolder;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+public class BankingDataProcessTest {
+    BankingDataProcess processToTest;
+    AccountHolder knownUser;
+
+    @BeforeEach
+    public void setup(){
+        knownUser = new AccountHolder("1402110922112412");
+        processToTest = new BankingDataProcess();
+    }
+
+    @Test
+    public void testGetDeposits() {
+        assertEquals(0.0F, processToTest.getDeposits(knownUser.getAccountNumber()).get(0));
+        assertEquals(2750.0F, processToTest.getDeposits(knownUser.getAccountNumber()).get(1));
+    }
+
+    @Test
+    public void testGetWithdrawals() {
+        assertEquals(1995.0F, processToTest.getWithdrawals(knownUser.getAccountNumber()).get(0));
```

Comment:
why not just have it equal to an array (like, the actual result) rather than testing each element? this is okay though if the array is too long

Reply:
Yeah the arrayy is over 2000 characters long and would be kind of ridiculous to test that's the only reason